geode-core/src/main/java/org/apache/geode/internal/logging/LoggingSession.java

```diff
@@ -83,7 +83,8 @@ public synchronized void startSession() {
     }
     if (logConfiguration) {
       String configInfo = configuration.getLogConfigSupplier().getLogConfig().toLoggerString();
-      logger.info(STARTUP_CONFIGURATION + configInfo);
+      logger.info(STARTUP_CONFIGURATION);
+      logger.info(configInfo);
```

Comment:
This code change will move the configInfo to an entirely new log line with a different log header.

The existing output looks like (see how the `### GemFire` is indented a tiny bit too far):
```
[info 2017/03/06 14:29:30.003 PST loc1 <main> tid=0x1] Startup Configuration:
   ### GemFire Properties defined with system property ###
  enable-cluster-configuration=true
  load-cluster-configuration-from-dir=false
  ### GemFire Properties defined with api ###
```
Your change will result in:
```
[info 2017/03/06 14:29:30.003 PST loc1 <main> tid=0x1] Startup Configuration:
[info 2017/03/06 14:29:30.014 PST loc1 <main> tid=0x1] ### GemFire Properties defined with system property ###
  enable-cluster-configuration=true
```
But the correct fix for the ticket is to maintain it as a single log statement and just fix the indentation so that `### GemFire` is lined up with the other lines:
```
[info 2017/03/06 14:29:30.003 PST loc1 <main> tid=0x1] Startup Configuration:
  ### GemFire Properties defined with system property ###
  enable-cluster-configuration=true
  load-cluster-configuration-from-dir=false
  ### GemFire Properties defined with api ###
```
The fix probably requires a change to AbstractConfig or DistributionConfigImpl instead of LoggingSession. LoggingSession should not have to know how to format configInfo which is provided by the Config.

Reply:
Thats right, I have not realized Im introducing a new log header. The tests dont detect that, as they only look for `STARTUP_CONFIGURATION` and `configInfo`.

What about including a line separator in the string?
```
logger.info(STARTUP_CONFIGURATION + lineSeparator() + configInfo);
```
I have just tested and the result is the expected:

```
[info 2019/05/29 07:29:03.813 GMT <main> tid=0x1] Startup Configuration:
### GemFire Properties defined with system property ###
start-dev-rest-api=false                                                 
use-cluster-configuration=true                                            
### GemFire Properties defined with api ###        
name=server1

```

Reply:
That's probably ok if the resulting lines line up fine (looks like they do).